src/constants.js

```diff
@@ -9,3 +9,7 @@ export const USER_INTERFACE_ID = 'user-interface';
 export const START_QUIZ_BUTTON_ID = 'start-quiz-button';
 export const ANSWERS_LIST_ID = 'answers-list';
 export const NEXT_QUESTION_BUTTON_ID = 'next-question-button';
+export const CONFIRM_BUTTON = 'confirm-button';
```

Comment:
- Variable names should be updated like below (you need to add "ID" at the end of each variable name):
```
export const CONFIRM_BUTTON_ID = 'confirm-button';
export const INPUT_NAME_ID = 'player-name-input';
export const INPUT_P_TAG_ID = 'name-prompt';
export const WELCOME_MESSAGE_ID = 'welcome-message';
```